Analytics/Request/BlockingRequestHandler.cs

```diff
@@ -243,7 +243,7 @@ internal BlockingRequestHandler(Client client, TimeSpan timeout)
 							// If status code is greater than 500 and less than 600, it indicates server error
 							// Error code 429 indicates rate limited.
 							// Retry uploading in these cases.
-							Task.Delay(backoff).Wait();
+							await Task.Delay(backoff);
```

Comment:
is there a mimimum version of .net this must be used with? wondering if there's a chance of breaking compatibility?